skt/executable.py

```diff
@@ -44,6 +44,15 @@
 retcode = SKT_SUCCESS
 
 
+class CustomAction(argparse.Action):
```

Comment:
I'd use a more descriptive class name, like `AppendMergeParameter` (or something shorter, but the idea is to have a descriptive name)

Reply:
What about the name `AppendMergeArgument` ?

Reply:
Sure, that works too